components/BottomNavigation/src/MDCBottomNavigationBarController.m

```diff
@@ -56,6 +87,17 @@ - (void)viewDidLoad {
   [self.view addSubview:self.content];
   [self.view addSubview:self.navigationBar];
   [self loadConstraints];
+
+  if ([self isDynamicTypeSupportEnabled] && ![self isNavigationBarLongPressRecognizerRegistered]) {
```

Comment:
I am following the [Google objective-c style](http://google.github.io/styleguide/objcguide.html#objective-c-method-names) guide here.  If I have misinterpreted these rules or if this project uses a different style, please let me know.

```
BOOL isGood = [object isGlorious];  // GOOD.
BOOL isGood = object.glorious;      // GOOD.
BOOL isGood = object.isGlorious;    // AVOID.
```

Reply:
Agreed for method names. Since `isDynamicTypeSupportEnabled` is declared as a property, it should be accessed like a property.

```objc
@property(nonatomic, getter=isDynamicTypeSupportEnabled) BOOL dynamicTypeSupportEnabled;
```

Every property is implemented with a method, I believe the style guide refers to more traditional methods (that are not declared as properties).

Reply:
Done.